install_files/ansible-base/roles/noble-migration/tasks/main.yml

```
@@ -0,0 +1,92 @@
+---
+
+- name: Check migration JSON on mon server
```

This is all host-agnostic, no?

```suggestion
- name: Check host's migration state
```

---

Yep, thanks. Leftover from before my refactoring.